README.md

```diff
@@ -280,10 +280,10 @@ Julia uses the following external libraries, which are automatically downloaded
 - **[SuiteSparse]** (>= 4.1) — library of linear algebra routines for sparse matrices.
 - **[ARPACK]**               — collection of subroutines designed to solve large, sparse eigenvalue problems.
 - **[FFTW]** (>= 3.3)        — library for computing fast Fourier transforms very quickly and efficiently.
-- **[PCRE]** (>= 10.00)      — Perl-compatible regular expressions library.
+- **[PCRE2]** (>= 10.00)     — Perl-compatible regular expressions library.
 - **[GMP]** (>= 5.0)         — GNU multiple precision arithmetic library, needed for `BigInt` support.
 - **[MPFR]** (>= 3.0)        — GNU multiple precision floating point library, needed for arbitrary precision floating point (`BigFloat`) support.
-- **[libgit2]** (>= 0.23)    — Git linkable library, used by Julia's package manager
+- **[libgit2]** (>= 0.23)    — a permissively linkable Git library, used by Julia's package manager
```

Comment:
And "permissive" is typically used to describe MIT/BSD-style licenses, not LGPL or GPL with linking exception.